Phase01Project1/src/InvertedIndex.java

```diff
@@ -0,0 +1,48 @@
+import Documents.Document;
+import Normalizer.TokenNormalization;
+import Tokenize.Tokenizer;
+
+import java.util.*;
+
+public class InvertedIndex {
+
+    private ArrayList<Document> documents;
+    private Map<String, ArrayList<Document>> wordDocuments;
```

Comment:
If there is an input of method with polymorphed object, then it's better to use an interface. but when it is an internal thing for class, type must be fixed. Map X -> HashMap \/